android/res/values-zh-rTW/strings.xml

```diff
@@ -817,8 +817,10 @@
 	<string name="type.barrier.block">街區</string>
 	<string name="type.barrier.bollard">柱</string>
 	<string name="type.barrier.border_control">邊境管制</string>
+	<string name="type.barrier.ditch">護城河</string>
```

Comment:
@s8321414 maybe you need ask @rtsisyk add you to [translation-zh](https://github.com/orgs/organicmaps/teams/translations-zh) mentioned in #1489